kornia/__init__.py

```diff
@@ -14,3 +11,13 @@
 from kornia.constants import pi
 from kornia.testing import xla_is_available
 from kornia.utils import eye_like, vec_like, create_meshgrid, image_to_tensor, tensor_to_image
+
+# Version variable
+import sys
+
+if sys.version_info >= (3, 8):  # pragma: >=3.8 cover
+    import importlib.metadata as importlib_metadata
+else:  # pragma: <3.8 cover
+    import importlib_metadata
+
+__version__ = importlib_metadata.version('kornia')
```

Comment:
for this one, i recently discovered `version = attr:` in a separate project i'm working on
[version = attr:](https://github.com/farm-ng/amiga-dev-kit/blob/main/brain/setup.cfg#L3)

less code but i kinda feel that having in setup.cfg seems more direct and find to easy. Any opinions ?

Reply:
Personally, I prefer it directly in setup.cfg because that's where you would expect it to be

Reply:
sounds good to me